pkg/profiles/openapi.go

```diff
@@ -104,11 +106,13 @@ func swaggerToServiceProfile(swagger spec.Swagger, namespace, name, clusterDomai
 	return profile
 }
 
-func mkRouteSpec(path, pathRegex string, method string, responses *spec.Responses) *sp.RouteSpec {
+func mkRouteSpec(path, pathRegex string, method string, responses *spec.Responses, exception *spec.Operation) *sp.RouteSpec {
+	xLinkerdRetryable, _ := exception.VendorExtensible.Extensions.GetBool(xLinkerdRetryable)
 	return &sp.RouteSpec{
 		Name:            fmt.Sprintf("%s %s", method, path),
 		Condition:       toReqMatch(pathRegex, method),
 		ResponseClasses: toRspClasses(responses),
+		IsRetryable:     xLinkerdRetryable,
 	}
```

Comment:
Should this instead be?
```suggestion
func mkRouteSpec(path, pathRegex string, method string, responses *spec.Responses, exception *spec.Operation) *sp.RouteSpec {
	retryable := false
	if exception != nil {
		retryable, _ = exception.VendorExtensible.Extensions.GetBool(xLinkerdRetryable)
	}
	return &sp.RouteSpec{
		Name:            fmt.Sprintf("%s %s", method, path),
		Condition:       toReqMatch(pathRegex, method),
		ResponseClasses: toRspClasses(responses),
		IsRetryable:     retryable,
	}
}
```

Reply:
@Pothulapati I realised the same. :) Thank you very much.